### ISSUE TITLE
Please look my problem 

### ISSUE THREAD

**Issue by sathishmass** [isaumya/adsense-invalid-click-protector#4]
Bro mark my words I'm not joking. I have using the aicp plugin for 2 years and I well knew to configure the plugin. This week at the same time all of my websites the aicp plugin isn't working not blocking the ads too and i  checked some nulled paid plugin that works fine. Only in ur plugin, there is a bug.

SIR it's MY REQUEST PLEASE LOOK AT UR CODE AND ALSO PLEASE TEST THE PUGIN SIR. This problem occurs most of the Wordpress users. 

please sir....

**Comment by isaumya** [comment]
Do you have a test site where I can check this matter?

**Comment by kiranraj1430** [comment]
Yes in this website i have enabled 2 ads inside the post check it

**Comment by isaumya** [comment]
I don't see any ads on your site: https://i.imgur.com/8qwVzlT.png

**Comment by sathishmass** [comment]
![image](https://user-images.githubusercontent.com/63893887/79639505-e3d40300-81a9-11ea-810a-42754faee914.png)
 please check it again the ads have been showing perfectly... please go to the post of the website, not the home page. 

due to aicp problem, I have disabled the ads on the home page. kindly please check inside in the post content

**Comment by isaumya** [comment]
Can you disable the plush notification plugin and see if it works?

**Comment by isaumya** [comment]
I need to check this in detail. Currently, I'm very busy, as soon as I get some tile, will test this through and push an update.

**Comment by sathishmass** [comment]
sir this plugin isn't working in the past few days only, before that it's working perfectly. 

I have disabled all plugin but I won't work at all. I tried in all my 5 sites same problem occurs i have clicked my ads 6 times but still it showing. 

for my words please check and test ur awesome plugin. 

yesterday 50k page views only 4 IP blocked today 16k page views only 6ip blocked and my AdSense ctr is above 10%. 
![image](https://user-images.githubusercontent.com/63893887/79639826-b0927380-81ab-11ea-90f4-62cbceec74c9.png)

![image](https://user-images.githubusercontent.com/63893887/79639840-c99b2480-81ab-11ea-9527-5b830a640674.png)

day by day the IPs are reducing comparing to pageviews and i have noted that!. Then yesterday and today the plugin is not working 90%. 

ill promise there are some bugs in ur plugin. please test and release the plugin. you can also release the plugin in the premium version we will pay it also but aicp is very important for our future success.

**Comment by isaumya** [comment]
Hi,
I will definitely check and test the plugin when I get some time. I have some family emergency so can't look into it on an immediate basis.

**Comment by sathishmass** [comment]
> I need to check this in detail. Currently, I'm very busy, as soon as I get some tile, will test this through and push an update.

we will also contribute 20$ for u but don't leave bro because we are totally disappointed about this issue in corona time. we are only earning in the blogging. please, sir.we are only based on adsense earnings 

its my humble request to u

**Comment by isaumya** [comment]
@sathishmass just made some adjustments to the plugin, can you quickly check if this is working perfectly or not? I have tested here. WOrking perfectly. But still, test and let me know.

[ad-invalid-click-protector.zip](https://github.com/isaumya/adsense-invalid-click-protector/files/4496897/ad-invalid-click-protector.zip)

**Comment by isaumya** [comment]
@sathishmass Just release v1.2.4. Please download the update from wp-admin and test. Let me know if it works on your end. I've tested it on my end and it is working perfectly.

**Comment by kiranraj1430** [comment]
> @sathishmass Just release v1.2.4. Please download the update from wp-admin and test. Let me know if it works on your end. I've tested it on my end and it is working perfectly.

ok dude we will check that

**Comment by kiranraj1430** [comment]
Since 19th April morning at 12.42 Pm the plugin works perfectly instead of this I'll check again in the early morning because at that time we will get some more traffic

**Comment by kiranraj1430** [comment]
Yeah, we have tested completely. it has working on desktop perfectly but average in mobile phones. 

out of 100 / 70 percentage only blocking in this newer version. Some ips aren't detecting.

Comparing to last version it is doing its work super but  90% only. If you have a free time please note this issue. some ips are coming repeatedly after its blocked. 

I have clicked 6 times but not works after some time it blocked. now it is a little bit great but don't forget to fix the issue completely sir.

Thank you

**Comment by isaumya** [comment]
Please read the plugin description on the WordPress.org plugin page. I've updated the info there regarding mobile devices already.

**Comment by kiranraj1430** [comment]
> Please read the plugin description on the WordPress.org plugin page. I've updated the info there regarding mobile devices already.

But most of the website only receives 99% mobile tariff only... your last version works perfectly in mobile device also dude

**Comment by kiranraj1430** [comment]
![image](https://user-images.githubusercontent.com/63893887/79679290-508ae400-8222-11ea-94e9-6f9ea225d57a.png)
 you can look at the screenshot. one IP has been clicking my ads more than 208 times but it was blocking. the last version blocks the IP in mobile devices perfectly.

**Comment by isaumya** [comment]
@sathishmass are you using the latest version? How can one IP show up multiple times? Also, the plugin will not work effectively on mobile.

> **Important Note About Touch Screen Support:** This plugin doesn’t work on mobile devices such as smartphones and tablets, because this hardware uses a touchscreen instead of a mouse as click input. This design makes the boundary monitoring trick ineffective.

**Comment by kiranraj1430** [comment]
that's the issue ur old version works perfectly in mobile devices too.

**Comment by isaumya** [comment]
@sathishmass The old version also didn't work on mobile 100% of the times. Sometimes it worked and most times it did not.
